src/content/getusermedia/record/js/main.js

```diff
@@ -40,23 +45,11 @@ recordButton.addEventListener('click', () => {
 const playButton = document.querySelector('button#play');
 playButton.addEventListener('click', () => {
   const superBuffer = new Blob(recordedBlobs, {type: 'video/webm'});
+  recordedVideo.src = null;
+  recordedVideo.srcObject = null;
   recordedVideo.src = window.URL.createObjectURL(superBuffer);
-  // workaround for non-seekable video taken from
```

Comment:
non-seekable videos are still a problem i think...

Reply:
Yep. But it doesn't affect this sample as you cannot seek in the remote video.

Reply:
right, because you removed the controls. It would be still good to have this as a place to point people to if they have questions about seeking. Unless we can get @yellowdoge to write up a lengthy blog post instead :-)

Reply:
Since it isn't directly concerning WebRTC I'm hesitant to keep it there. I would also like to avoid having ugly workarounds in the samples as far as possible. Better then to have a sample that we "disable" until it works.

Reply:
it is much more effective to keep this here and have people find it in context than having to find this hidden in come chrome bug. mediarecorder files won't be as easily seekable by design.

Reality is messy. WebRTC in particular so... and its getting worse actually.

Reply:
I'm leaving the comment out for now and I'm adding it to my list of "workaround snippets" that we can hopefully publish somewhere.